docs: Fix typo in Hue integration documentation (#41035)
home-assistant/home-assistant.github.io: source/_integrations/hue.markdown

Philips/Signify released a new version of their Hue bridge (square shape) and their legacy/V1 bridge (round shape) is now end of life and no longer supported by them. Home Assistant will continue to support the V1 Hue bridge as long as it is technically possible, although with a few limitations:

> - Scene entities are not automatically created for V1 bridges. To activate a Hue scene on a V1 bridge from Home Assistant we provide an action to active the Hue scene by name.

Reviewer: ```suggestion
- Scene entities are not automatically created for V1 bridges. To activate a Hue scene on a V1 bridge from Home Assistant, there's an action that allows you to activate the Hue scene by name.
```
